README.md

```diff
@@ -2,10 +2,9 @@
 Bowling Challenge
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+## Screenshot of my Scorecard interface
+
+![My scorecard](images/bowling-scorecard.png)
```

Comment:
Nice screenshot showing your project. I always love how you make the front end of your projects look awesome 👍